vernac/classes.ml

```diff
@@ -152,6 +152,28 @@ let default_locality () =
 let instance_locality =
   Attributes.hint_locality ~default:default_locality
 
+type event =
+  | NewClass of typeclass
+  | NewInstance of {
+      class_name : GlobRef.t;
+      instance : GlobRef.t;
+      info : Typeclasses.hint_info;
+      locality : Hints.hint_locality;
+    }
+
+let observers = Summary.ref ~name:"classes_observer" CString.Map.empty
```

Comment:
not marshallable

Reply:
Right. My conclusion was in the end that we don't want `Summary` at all, not libobjects.
We just want that dynlinked code sets these.